Doc/library/turtle.rst

```diff
@@ -14,6 +14,11 @@
    from turtle import *
    turtle = Turtle()
 
+.. testcleanup::
+
+   import os
+   os.remove("my_drawing.ps")
```

Comment:
Same suggestion as on the 3.13 PR:

```suggestion
   # Destroy the turtle window after tests are complete
   # Imported via star import in testsetup
   bye()
```

Reply:
Will leave this for now, same rationale as https://github.com/python/cpython/pull/125290#discussion_r1796707576